src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ObjectionStatus.java

```diff
@@ -5,5 +5,10 @@
     PROCESSED,
     SUBMITTED,
     INELIGIBLE_COMPANY_STRUCK_OFF,
-    INELIGIBLE_NO_DISSOLUTION_ACTION
+    INELIGIBLE_NO_DISSOLUTION_ACTION;
+
+    public boolean isEligibilityError() {
+        return this == INELIGIBLE_NO_DISSOLUTION_ACTION ||
```

Comment:
This new function needs a unit test, need to add an ObjectionStatusTest class

Reply:
Three new unit tests